ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -126,6 +128,7 @@ function ViolationsByPolicyCategoryChart({ alertGroups }: ViolationsByPolicyCate
             <Chart
                 ariaDesc="Number of violation by policy category, grouped by severity"
                 ariaTitle="Policy Violations by Category"
+                animate={{ duration: 300 }}
```

Comment:
I think this looks good, and makes it more clear when data is changing, but I can run it by UX during our Monday meeting.